.travis.yml

```diff
@@ -110,4 +110,3 @@ jobs:
       env: WP_VERSION=latest SWITCH_TO_PHP=5.2
       script:
         - ./bin/run-wp-unit-tests.sh
-
```

Comment:
Looks like this change is not needed?

Reply:
This change does not exist in my local machine. I do not understand why this appears.